prt_partner_tweaks_access/models/ir_rule.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import api, models
+
+from ..consts import PREDEFINED_RULES
+
+
+class IRRule(models.Model):
+    _inherit = "ir.rule"
+
+    @api.model
+    def tweak_access_rules(self, state):
```

Comment:
Docstring is incomplete.
And what is this function used for?

Reply:
okay, I'll fix it